src/couch_index_server.erl

```diff
@@ -38,7 +38,6 @@
 -record(st, {root_dir}).
 
 start_link() ->
-    couch_epi:register_service(couch_index),
```

Comment:
I'm glad you brought your point. There is actually a little problem. couch_epi need to be listed in dependencies of couch_index.